app/controllers/users.js

```diff
@@ -34,10 +34,28 @@ exports.signIn = (req, res, next) => {
     .then(foundUserPassword => comparePasswords(password, foundUserPassword))
     .then(result => {
       if (result) {
-        res.status(200).send({ token: createToken(email) });
+        res.status(200).send(createToken(email));
       } else {
         throw validationError('Password does not match with the email');
       }
     })
     .catch(next);
 };
+
+exports.listUsers = (req, res, next) => {
+  const { token, offset, limit } = req.query;
+  return validateToken(token)
+    .then(validated => {
+      if (validated) {
+        logger.info('User validated.\nGetting all users for listing them');
+        User.getUsers(offset, limit)
+          .then(foundUsers => {
+            res.status(200).send(foundUsers);
```

Comment:
While it is failry obvious here, it might not often be clear what you are responding with. Responses should be almost always wrapped in an object to give these responses a name, for example
`{users: foundUsers}`